RiotShareExtension/BuildSettings.swift

```diff
@@ -242,7 +242,7 @@ final class BuildSettings: NSObject {
     /// Indicates should the app log out the user when number of biometrics failures reaches `maxAllowedNumberOfBiometricsFailures`. Defaults to `false`
     static let logOutUserWhenBiometricsFailuresExceeded: Bool = false
     
-    static let showNotificationsV2: Bool = false
+    static let showNotificationsV2: Bool = true
```

Comment:
same remark

Reply:
It's defined in Element Share Extension target too.
And after the next rebase, there will be only one BuildSettings.swift file for all targets.